src/components/List.js

```diff
@@ -52,15 +70,15 @@ const List = ({ firestore }) => {
   };
 
   return (
-    <>
+    <Fragment>
```

Comment:
With a `<Fragment>` wapping the whole return, are the nested ones necessary? I’d be curious whether everything still renders if you delete all of the nested `<></>` tags.

Reply:
Just tried pulling out all of the nested <></> tags. No go. I get 'Adjacent JSX elements must be wrapped in an enclosing tag.' errors. 

I feel like this component is ready for some refactoring. It's kind of hard to read at the moment.

Reply:
OK, yeah, I agree that a refactor might be in order. Carry on! 🙂